src/System.Data.Common/tests/System/Data/DataSetTypedDataSetTest.cs

```diff
@@ -1127,159 +1066,159 @@ public IEnumerator GetEnumerator()
                     }
                 }
 
                 protected override void OnRowChanging(DataRowChangeEventArgs e)
                 {
                     base.OnRowChanging(e);
-                    if ((Order_DetailsRowChanging != null))
+                    if ((OrderDetailsRowChanging != null))
                     {
-                        Order_DetailsRowChanging(this, new Order_DetailsRowChangeEvent(((Order_DetailsRow)(e.Row)), e.Action));
+                        OrderDetailsRowChanging(this, new OrderDetailsRowChangeEvent(((OrderDetailsRow)(e.Row)), e.Action));
                     }
                 }
 
                 protected override void OnRowDeleted(DataRowChangeEventArgs e)
                 {
                     base.OnRowDeleted(e);
-                    if ((Order_DetailsRowDeleted != null))
+                    if ((OrderDetailsRowDeleted != null))
                     {
-                        Order_DetailsRowDeleted(this, new Order_DetailsRowChangeEvent(((Order_DetailsRow)(e.Row)), e.Action));
+                        OrderDetailsRowDeleted(this, new OrderDetailsRowChangeEvent(((OrderDetailsRow)(e.Row)), e.Action));
                     }
                 }
 
                 protected override void OnRowDeleting(DataRowChangeEventArgs e)
                 {
                     base.OnRowDeleting(e);
-                    if ((Order_DetailsRowDeleting != null))
+                    if ((OrderDetailsRowDeleting != null))
                     {
-                        Order_DetailsRowDeleting(this, new Order_DetailsRowChangeEvent(((Order_DetailsRow)(e.Row)), e.Action));
+                        OrderDetailsRowDeleting(this, new OrderDetailsRowChangeEvent(((OrderDetailsRow)(e.Row)), e.Action));
                     }
                 }
 
-                public void RemoveOrder_DetailsRow(Order_DetailsRow row)
+                public void RemoveOrder_DetailsRow(OrderDetailsRow row)
                 {
                     Rows.Remove(row);
                 }
             }
 
-            public class Order_DetailsRow : DataRow
+            public class OrderDetailsRow : DataRow
             {
-                private Order_DetailsDataTable _tableOrder_Details;
+                private OrderDetailsDataTable _tableOrderDetails;
 
-                internal Order_DetailsRow(DataRowBuilder rb) :
+                internal OrderDetailsRow(DataRowBuilder rb) :
                     base(rb)
                 {
-                    _tableOrder_Details = ((Order_DetailsDataTable)(Table));
+                    _tableOrderDetails = ((OrderDetailsDataTable)(Table));
                 }
 
                 public int OrderID
                 {
                     get
                     {
-                        return ((int)(this[_tableOrder_Details.OrderIDColumn]));
+                        return ((int)(this[_tableOrderDetails.OrderIDColumn]));
```

Comment:
Expression-bodied (throughout)

Reply:
These could certainly be changed.  But as I think you commented separately, this PR has already grown very big.  Let's leave any additional clean-up for a subsequent PR.